book/website/collaboration/inclusive-events/inclusive-events-resources.md

```diff
@@ -20,5 +21,92 @@
 (cl-inclusive-events-resources-reading)=
 ## Further Reading
 
+* [The 7 Principles of Universal Design](https://universaldesign.ie/about-universal-design/the-7-principles)
+* (Book) [The Art of Gathering: How We Meet and Why It Matters](https://www.priyaparker.com/book-art-of-gathering) by Priya Parker
+
+(cl-inclusive-events-resources-other)=
+## Additional Resources 
+
+This chapter cannot (and should not) replace working with members of your specific community to plan and host events. 
```

Comment:
```suggestion
These resources cannot (and should not) replace working with members of your specific community to plan and host events. 
```

Of course also true for the chapter at large, but I guess at this particular space mostly about the resources linked below?